internal/accounts/create.go

```diff
@@ -36,8 +36,8 @@ type flagsCreate struct {
 	Signer    string   `default:"emulator-account" flag:"signer" info:"Account name from configuration used to sign the transaction"`
 	Keys      []string `flag:"key" info:"Public keys to attach to account"`
 	Weights   []int    `flag:"key-weight" info:"Weight for the key"`
-	SigAlgo   string   `default:"ECDSA_P256" flag:"sig-algo" info:"Signature algorithm used to generate the keys"`
-	HashAlgo  string   `default:"SHA3_256" flag:"hash-algo" info:"Hash used for the digest"`
```

Comment:
We need to be careful with backward compatibility when we change flags. I strongly advise we have a check to see if only single flag was provided for multiple keys and still then set all those algos to that provided value but also output a warning that usage is deprecated. I would say to do that in every case where `len(sigAlgo) == 1 && len(keys) > 1`. We also need to be mindful about changing default values, since we potentially will remove default value we need to for a while still provide default values but also provide a deprecation notice. 
An example of such deprecation notices is:
https://github.com/onflow/flow-cli/blob/13897338d416f22bc8bf8f3805a9217d7d039d92/internal/keys/generate.go#L52